geomstats/geometry/spd_matrices.py

```diff
@@ -22,12 +22,14 @@ def __init__(self, n):
             embedding_manifold=GeneralLinear(n=n))
         self.n = n
 
-    def belongs(self, mat, atol=TOLERANCE):
-        """Check if a matrix belongs to the manifold.
-
-        Check if a matrix belongs to the manifold of symmetric positive
-        definite matrices.
-        """
+    @staticmethod
+    def belongs(mat, atol=TOLERANCE):
+        """Check if a matrix is symmetric and invertible."""
+        # FIXME: check positivity, implying invertibility (Yann)
+        #
+        # note : vectorized "and" on numpy works with:
+        #       [bool] * [bool] -> bool
+        # but does not on tf.
```

Comment:
What do you recommend there? We'll tackle this with decorators?

Reply:
We should probably provide a `gs.and` and `gs.or` in the backend for these kind of tests...
Actually I was surprised not to find those in numpy until I realized that `+` and `*` did the job!  

Only thing is to figure how tf and torch do this: if they really don't like booleans, we'll probably have to return 1. and 0. floats...